.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
By using the `eslint --init` to create your file initially and selecting the options to answer questions about your configuration file, you'll get the option to specify an environment. By specifying "browser" for your environment, you'll negate the need to add "window" to your globals.